mappings/net/minecraft/entity/passive/camel/CamelBrain.mapping

```diff
@@ -0,0 +1,23 @@
+CLASS net/minecraft/unmapped/C_hfncvwem net/minecraft/entity/passive/camel/CamelBrain
```

Comment:
Shouldn't this be in `entity/ai/brain`?

Reply:
No, its should be up one directory. However i want to move the brain/entity class combos into their own packages. Ill make another pr for that